.drone.yml

```diff
@@ -36,149 +36,6 @@ steps:
     event:
     - tag
 
-- name: docker-publish
```

Comment:
Should this be kept?

Reply:
no, as this binary will go in k3s-upgrade image, so we dont need a standalone image for it